WPMedia/fusion-news-theme-blocks
# PEN-1270 alert bar dismiss update

[PEN-1270](https://arcpublishing.atlassian.net/browse/PEN-1270)

# What does this implement or fix?
- fix a case when the user load the page (no cookie exists), the alert bar render, the user dismiss it, and on the next refresh alert cycle, the alert bar appear again.

# How was this tested?
- visually on PB

To test it:
- clear all the cookies or the one named `arcblock_alert-bar`
- load a page with an alert bar
- dismiss it when page finish loading
- wait 2 minutes for an alert bar refresh cycle (can check the network tab to view when the request is done)
- the alert bar must not appear again.

- check the same, but changing the story on the alert bar collection after dismiss it.

# Dependencies or Side Effects

- none
